src/servers/authorization/index.ts

```diff
@@ -189,14 +193,19 @@ authRouter.post("/create/device-account", async (req: Request, res: Response) =>
 
   const username = user.name
   const password = user.pass
-  const deviceId = username
+  const deviceIdRaw: string = username
```

Comment:
seems like we just use the username as the `deviceIdRaw` not sure if it makes more sense just to use `username` on line 208 instead of creating a new variable here